content/about/community/partners/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Partners
+description: Who's building stuff around Istio.
+weight: 20
+page_icon: /img/partners.svg
+keywords: [community]
+skip_seealso: true
+---
+
+Istio benefits from an ecosystem of partners which help make Istio great. Check 'em out and see how they make Istio even better.
```

Comment:
Istio benefits from a great ecosystem of partners helping make Istio great.  Check 'em out and learn how they make Istio even better.